hazelcast/src/main/resources/cluster.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
    	echo "parameters : "
-   	echo "	-o, --operation	    : Defines state operation of the script. Operation can be 'get' or 'change'."
+   	echo "	-o, --operation	    : Defines state operation of the script. Operation can be 'get','change','shutdown','force-start'."
     echo "	-s, --state 	    : Defines new state of the cluster. State can be 'active', 'frozen', 'passive'"
```

Comment:
`Defines new state of the cluster` --> `updates cluster state` or something similar.